examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,242 @@ many cases it is best to combine both approaches. The file can be easily
 reloaded by pressing "reload" under the "Geometry" tab if you want to write
 it by hand and see the effects in the graphical user interface of gmsh.
 
+
+
 This tutorial contains an example <code>.geo</code> file that describes a box
-with two objects cut out in the interior. This is how
+with two circles cut out in the interior and several slits on the sides. This is how
 <code>example.geo</code> looks like in %Gmsh (displaying the boundary
 indicators as well as the mesh discussed further down below):
 
-<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture.png" alt="">
+<img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture2.png" alt="">
+
+We will go through the basics of %Gmsh and show how we can obtain a mesh like this.
+
+It's also worth bringing up this video, https://www.youtube.com/watch?v=xL2LmDsDLYw&t=921s,
+(I don't know how to hyperlink here)
```

Comment:
You can look at other tutorials (e.g. step-68,step-70 and many others) where hyperlinks are embedded :)